src/components/Main.ui.js

```diff
@@ -199,498 +190,511 @@ export class MainComponent extends Component<Props> {
               renderLeftButton={this.renderBackButton()}
               renderRightButton={this.renderHelpButton()}
             />
-
-            <Drawer key={Constants.EDGE} hideNavBar contentComponent={ControlPanel} hideDrawerButton={true} drawerPosition="right" drawerWidth={scale(280)}>
-              {/* Wrapper Scene needed to fix a bug where the tabs would reload as a modal ontop of itself */}
-              <Scene key={'AllMyTabs'} hideNavBar>
-                <Tabs
-                  key={Constants.EDGE}
-                  swipeEnabled={false}
-                  navTransparent={true}
-                  tabBarPosition={'bottom'}
-                  showLabel={true}
-                  tabBarStyle={styles.footerTabStyles}
-                >
-                  <Stack key={Constants.WALLET_LIST} icon={this.icon(Constants.WALLET_LIST)} tabBarLabel={WALLETS}>
+            {this.props.isLoggedOut && (
```

Comment:
There is too much indentation happening here, and has been for a while. Now is a good time to fix it, instead of making it worse!

Please move this block of code into its own method, `renderMainApp` or something, and then this line can be `{this.props.isLoggedOut && this.renderMainApp() }`.